packages/node/src/node/ClientNode.ts

```diff
@@ -110,22 +116,42 @@ export class ClientNode extends Node<ClientNode.RootEndpoint> {
 
     /**
      * Remove this node from the fabric (if commissioned) and locally.
+     * This method tries to communicate with the device to decommission it properly and will fail if the device is
+     * unreachable.
+     * If you can not reach the device, use {@link delete} instead.
      */
-    override async delete() {
+    async decommission() {
         if (this.lifecycle.isCommissioned) {
             this.statusUpdate("decommissioning");
 
             await this.act("decommission", agent => agent.commissioning.decommission());
         }
+        await this.delete();
+    }
 
-        await super.delete();
+    override async start() {
+        if (this.#startInProgress) {
```

Comment:
Skylar this be in Node instead?  It doesn't really seem like it's ClientNode specific.  It basically protects from the beginning of the call to the point where #runtime is set